src/main/webapp/mapScript.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+let map;
+
+function initMap(showMarkers = true) {
+  // Create markers.
+  let japanTemple = { lat: 34.462117, lng: 135.830272 };
+  let newZealandLake = { lat: -43.979931, lng: 170.194799 };
+  let ugandaView = { lat: -0.099273, lng: 32.652921 };
```

Comment:
I think it's better to have [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) than having hardcoded some coordinates.
Or you can pass these parameters as normal ones.

Reply:
Changed them to default.